client/blocks/plan-storage/bar.jsx

```diff
@@ -14,19 +13,10 @@ export class PlanStorageBar extends Component {
 		className: PropTypes.string,
 		mediaStorage: PropTypes.object,
 		displayUpgradeLink: PropTypes.bool,
-		sitePlanSlug: PropTypes.string.isRequired,
 	};
 
 	render() {
-		const { className, displayUpgradeLink, mediaStorage, sitePlanSlug, translate } = this.props;
-
-		if ( planHasFeature( sitePlanSlug, FEATURE_UNLIMITED_STORAGE ) ) {
-			return null;
-		}
```

Comment:
This check was already in place on the parent `PlanStorage` component:

https://github.com/Automattic/wp-calypso/blob/c428208ca12a454155fac66c4b4a6df1c0bde0ec/client/blocks/plan-storage/index.jsx#L53-L55